### DISCUSSION TITLE
How can I test a subscription using GraphiQL and DGS ?

### DISCUSSION THREAD

**Discussion by lthoulon-locala** [Netflix/dgs-framework#1643] [Q&A] [answered]
Hello. I'm trying to use GraphiQL to test my first subscription but I can't make it work. The only answer I get is the following:

```json
{
  "data": {
    "upstreamPublisher": {
      "scanAvailable": true,
      "prefetch": -1
    }
  }
}
```

It looks like GraphiQL is calling. the `/graphql` endpoint instead of the `/subscriptions` endpoint.
As far as I know, it should be supported by GraphiQL. See https://github.com/graphql/graphiql/issues/1486

Any idea if I'm missing something or if it's just not supported in the DGS version yet ?
Thanks

**Comment by srinivasankavitha** [comment, marked as answer]
@lthoulon-locala - we don't support this yet.

**Reply by lthoulon-locala** [reply]
Thanks. That's very clear.
Is it something already in an enhancement request backlog or should I create an issue to do so ?
Thank you.

**Reply by srinivasankavitha** [reply]
Please go ahead and create an issue. We will prioritize and work on it when we are able to get to it. Thanks for bringing this up.
